oceanwaters/doc/setup_dev_env.md

```diff
@@ -225,30 +223,31 @@ sudo apt-get upgrade
 sudo apt install git \
                  python-wstool \
                  python-catkin-tools \
-                 ros-melodic-tf2-ros \
-                 ros-melodic-robot-state-publisher \
-                 ros-melodic-joint-state-publisher \
-                 ros-melodic-joint-state-controller \
-                 ros-melodic-effort-controllers \
-                 ros-melodic-joint-trajectory-controller \
-                 ros-melodic-dynamic-reconfigure \
-                 ros-melodic-nodelet \
-                 ros-melodic-nodelet-topic-tools \
-                 ros-melodic-camera-info-manager \
-                 ros-melodic-tf2-geometry-msgs \
-                 ros-melodic-gazebo-ros-control \
-                 ros-melodic-xacro ros-melodic-rviz-visual-tools \
-                 ros-melodic-rqt-plot ros-melodic-rqt-rviz \
-                 ros-melodic-rqt-image-view \
-                 ros-melodic-rqt-common-plugins \
-                 ros-melodic-gazebo-plugins \
-                 ros-melodic-moveit \
-                 ros-melodic-moveit-ros-visualization \
-                 ros-melodic-geometry-msgs \
-                 ros-melodic-cmake-modules \
-                 ros-melodic-stereo-msgs \
-                 ros-melodic-stereo-image-proc \
-                 ros-melodic-kdl-parser-py \
+                 ros-noetic-tf2-ros \
+                 ros-noetic-robot-state-publisher \
+                 ros-noetic-joint-state-publisher \
+                 ros-noetic-joint-state-controller \
+                 ros-noetic-effort-controllers \
+                 ros-noetic-joint-trajectory-controller \
+                 ros-noetic-dynamic-reconfigure \
+                 ros-noetic-nodelet \
+                 ros-noetic-nodelet-topic-tools \
+                 ros-noetic-camera-info-manager \
+                 ros-noetic-tf2-geometry-msgs \
+                 ros-noetic-gazebo-ros-control \
+                 ros-noetic-xacro \
+								 ros-noetic-rviz-visual-tools \
```

Comment:
Looks like you accidentally added tabs instead of spaces.

Reply:
Oops - should be fixed now.